### PR TITLE
Update package path to /wish/ from /ContextLogic/

### PR DESCRIPTION
**Pull request by bretep**
This needs to be updated since moving the project from the `github.com/ContextLogic` to `github.com/wish`

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 [[projects]]
   branch = "master"
-  name = "github.com/ContextLogic/goServiceLookup"
```
Comment (tvi) [inline Gopkg.lock]: This package does not exists :( in both orgs

Comment (bretep) [reply Gopkg.lock]: I just search and replaced. Is `github.com/ContextLogic/goServiceLookup` an internal project?

Comment (tvi) [reply Gopkg.lock]: yep :(

Comment (bretep) [reply Gopkg.lock]: I'll leave that up to you guys to fix then. ;)

**Pull request closed without merging**